scipy/optimize/_linprog_rs.py

```diff
@@ -255,11 +258,76 @@ def _phase_two(c, A, x, b, maxiter, tol, maxupdate, mast):
 
 
 # FIXME: is maxiter for each phase?
```

Comment:
The documentation defines max iter as ``Maximum number of iterations to perform.``, so it's not *strictly* defined. ``linprog(method='simplex')``  interperates this as the number of iterations required for **both** phases not each individual phase.

Reply:
Not sure which I prefer. On one hand, that makes sense. But it's sort of unfair to phase 2 if phase 1 eats up more than its fair share of iterations. Suggestion?